### DISCUSSION TITLE
crazyflie deck

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1198] [Q&A]
Hi, Is it possible to install three decks on crazyflie? uwb deck, flow deck, and mocap marker deck. I thought the long pin is not enough to connect them. But I need to use vicon camera to track crazyflie in UWB navigation system. thanks

**Comment by gemenerik** [comment]
These decks should be [compatible](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#compatibility-matrixes), given you use the long pins.
